### PR TITLE
Doc: Update Dual DC L3LS example

### PR DESCRIPTION
**Pull request by xaviramon**
## Change Summary

Update of Dual DC example

## Proposed changes
Update README.md doc and group_vars

## How to test
Generate docs and validate REAMDE.md file and we follow best practices for group_vars

### CODE REVIEW

File: ansible_collections/arista/avd/examples/dual-dc-l3ls/documentation/devices/dc2-leaf1a.md [hunk 1]
```diff
@@ -56,7 +56,7 @@
 
 | Management Interface | description | Type | VRF | IP Address | Gateway |
 | -------------------- | ----------- | ---- | --- | ---------- | ------- |
-| Management1 | oob_management | oob | MGMT | 172.16.2.201/24 | 172.16.2.1 |
+| Management1 | oob_management | oob | MGMT | 172.16.2.201/24 | 172.16.1.1 |
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/examples/dual-dc-l3ls/documentation/devices/dc2-leaf1a.md]: this gateway looks wrong

File: ansible_collections/arista/avd/examples/dual-dc-l3ls/group_vars/DC2.yml [hunk 2]
```diff
@@ -14,17 +14,16 @@ spine:
...
-  # Specific configuration of device must take place under the node definition. Each node inherites all values defined under 'defaults'
+  # Specific configuration of device must take place under the node definition. Each node inherits all values defined under 'defaults'
   nodes:
     # Name of the node to be defined (must be consistent with definition in inventory)
     - name: dc2-spine1
       # Device ID definition. An integer number used for internal calculations (ie. IPv4 address of the loopback_ipv4_pool among others)
       id: 11
       # Management IP to be assigned to the management interface
-      mgmt_ip: 172.16.2.21/24
+      mgmt_ip: 172.16.1.21/24
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/examples/dual-dc-l3ls/group_vars/DC2.yml:26]: dc2 should be 172.16.2.x I believe.

Comment (JulioPDX) [reply ansible_collections/arista/avd/examples/dual-dc-l3ls/group_vars/DC2.yml:26]: You are correct, no idea what happened here. Ill get this fixed

Comment (JulioPDX) [reply ansible_collections/arista/avd/examples/dual-dc-l3ls/group_vars/DC2.yml:26]: @ClausHolbechArista I updated everything in the scenario to live on 172.16.1, if need be I can revert back to the original setup. The only reason I set everything to .1 was for lab creation simplicity

Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/examples/dual-dc-l3ls/group_vars/DC2.yml:26]: @xaviramon Do you have any strong opinions on using the same or different mgmt subnets for dc1 and dc2 in this example?

**Review by ClausHolbechArista — APPROVED** [review]

**Review by carlbuchmann — APPROVED** [review]
LGTM

**Pull request merged**